deployment/import_instructions.md

```diff
@@ -0,0 +1,114 @@
+# Import Instructions
+
+## From `pete-sls` branch:
+
+1. Deploy sls to get it ready for deletion with retained resources configured for import
+
+```
+./run deploy --stage <YOUR_BRANCH_NAME>
+```
+
+2. Collect information about the resources we're going to be importing into the new cdk stack.
+```
+cloudfront.Distribution -
+cognito.UserPool -
+```
+
+3. Destroy sls
+
+```
+./run destroy --stage <YOUR_BRANCH_NAME>
+```
+
+:warning: Make sure that all sls associated stacks have been fully destroyed before proceeding.
```

Comment:
Is there a way we can validate that the retain setting took before we destroy the stack?

Reply:
This was done as CMDCT-4238 which you did, thanks!